rocAL/source/api/rocal_api_data_loaders.cpp

```diff
@@ -34,6 +34,8 @@ THE SOFTWARE.
 #include "loaders/image/node_fused_jpeg_crop_single_shard.h"
 #include "loaders/image/node_image_loader.h"
 #include "loaders/image/node_image_loader_single_shard.h"
+#include "loaders/image/node_numpy_loader.h"
+#include "loaders/image/node_numpy_loader_single_shard.h"
 #ifdef ROCAL_AUDIO
```

Comment:
please move the #ifdef ROCAL_VIDEO here so they can be after the common include files